modules/reference/pages/topic-properties.adoc

```diff
@@ -242,6 +246,44 @@ When `segment.bytes` is set to a positive value, it overrides the cluster proper
 
 ---
 
+[[writecaching]]
+==== write.caching
+
+The write caching mode to apply to a topic. 
+
+When `write.caching` is set to on, it overrides the cluster property xref:cluster-properties.adoc#write_caching[`write_caching`]. This caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. Flushes follow <<flushms, `flush.ms`>> and <<flushbytes, `flush.bytes`>>, whichever is reached first. 
```

Comment:
remove reference to segment appender chusnk..

> a special mode of acks=all that doesn't fsync every message to disk for faster performance,

also lets use flush/fsync consistently, I'd vote for fsync.